great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -681,6 +681,7 @@ def run_profiler_on_data(
 
 
 def build_map_metric_rule(
+    data_assistant_class_name: str,
```

Comment:
Why is this relevant now?

Reply:
@NathanFarmer This is for making the error message informative.